SingularityUI/app/router.coffee

```diff
@@ -105,7 +105,7 @@ class Router extends Backbone.Router
         app.bootstrapController new DeployDetailController {requestId, deployId}
 
     aggregateTail: (requestId, path = '') ->
-        offset = window.location.hash.substr(1) || null
+        offset = parseInt(window.location.hash.substr(1)) || null
```

Comment:
Should pass a radix to `parseInt` to make sure its always parsed the same `parseInt(window.location.hash.substr(1),10)`